tests/Doctrine/Tests/ORM/Functional/ClassTableInheritanceTest.php

```diff
@@ -361,8 +355,9 @@ public function testQueryForInheritedSingleValuedAssociation()
         $this->_em->flush();
         $this->_em->clear();
 
-        $dql = "SELECT m FROM Doctrine\Tests\Models\Company\CompanyManager m WHERE m.spouse = ?1";
-        $dqlManager = $this->_em->createQuery($dql)->setParameter(1, $person->getId())->getSingleResult();
+        $dqlManager = $this->_em->createQuery('SELECT m FROM ' . CompanyManager::class . ' m WHERE m.spouse = ?1')
```

Comment:
I did it and it's not really a problem, I won't push it because I'm not sure about the conflict resolutions (`develop` conflicts with current state of `master` already). However I can help @guilhermeblanco if needed.